driver-adapters-wasm/d1-vitest/run.sh

```diff
@@ -0,0 +1,9 @@
+#!/usr/bin/env bash
+
+set -eu
+
+pnpm install
+pnpm prisma generate
+
+# Vitest is run locally, we don't need to deploy the database migration
+pnpm wrangler d1 migrations apply d1-vitest --local
```

Comment:
`d1 migrations apply` already defaults to local fwiw
https://developers.cloudflare.com/workers/wrangler/commands/#migrations-apply

Reply:
Sounds good to me :)